contracts/BSCValidatorSet.sol

```diff
@@ -114,8 +126,13 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
         validatorExtraSet.push(validatorExtra);
       }
     }
+    _;
+  }
 
+  modifier oncePerBlock() {
+    require(block.number > previousHeight, "can not distribute finality reward twice in one block");
```

Comment:
`can not distribute finality reward twice in one block` is not a proper message inside `oncePerBlock`, `oncePerBlock` is  a common modifier.

Reply:
Fixed